midi/drivers/alsa_midi.c

```diff
@@ -435,7 +435,7 @@ static bool alsa_midi_write(void *p, const midi_event_t *event)
    else if (ev.type == SND_SEQ_EVENT_PITCHBEND)
    {
       ev.data.control.channel = event->data[0] & 0x0F;
-      ev.data.control.value   = event->data[1] | (event->data[2] << 7);
+      ev.data.control.value   = (event->data[1] | (event->data[2] << 7)) - 0x2000;
```

Comment:
Seems like a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming))... Does this attribute for the pitch offset? Will it always be `0x2000`?